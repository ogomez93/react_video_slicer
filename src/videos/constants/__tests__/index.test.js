import {
  DEFAULT_VIDEO_URL,
  LOCAL_STORAGE_CLIPS_KEY,
  TIME_PRECISION
} from '../index';

describe('Videos / Constants / index', () => {
  describe('DEFAULT_VIDEO_URL', () => {
    it('should match the snapshot', () =>
      expect(DEFAULT_VIDEO_URL).toMatchSnapshot());
  });

  describe('LOCAL_STORAGE_CLIPS_KEY', () => {
    it('should match the snapshot', () =>
      expect(LOCAL_STORAGE_CLIPS_KEY).toMatchSnapshot());
  });

  describe('TIME_PRECISION', () => {
    it('should match the snapshot', () =>
      expect(TIME_PRECISION).toMatchSnapshot());
  });
});
