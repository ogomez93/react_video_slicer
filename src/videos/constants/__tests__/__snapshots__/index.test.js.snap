// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videos / Constants / index DEFAULT_VIDEO_URL should match the snapshot 1`] = `"https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"`;

exports[`Videos / Constants / index LOCAL_STORAGE_CLIPS_KEY should match the snapshot 1`] = `"videoSlicerClips"`;

exports[`Videos / Constants / index TIME_PRECISION should match the snapshot 1`] = `6`;
